pkg/component/component.go

```diff
@@ -321,3 +323,34 @@ func Exists(client *occlient.Client, componentName string) (bool, error) {
 	}
 	return false, nil
 }
+
+// Get Component Description
+func GetComponentDesc(client *occlient.Client, cmpnt string) (path string, componentURL string, appStore []storage.StorageInfo, err error) {
+	//Application
+	currentApplication, err := application.GetCurrent(client)
+	if err != nil {
+		return "", "", []storage.StorageInfo{}, errors.Wrap(err, "Unable to get current Application")
+	}
+	//Project
+	currentProject := project.GetCurrent(client)
```

Comment:
the same as with application. Get current project outside of this function and pass it as parameter